props/saber_BC_buttons.h

```diff
@@ -1115,6 +1126,10 @@ class SaberBCButtons : public PROP_INHERIT_PREFIX PropBase {
         return;
       // On-Demand Battery Level
       case EFFECT_BATTERY_LEVEL:
+        if (speaking_) {
+          speaking_ = false;
```

Comment:
Wouldn't it make more sense to say the battery level here?

Reply:
3 options can be used.
Spoken level in volts, spoken level in percentage, or On-Demand which is just showing the level on the blade with battery.wav sound played.
The way it's submitted is the intended usage.